projects/plugins/jetpack/extensions/extended-blocks/paid-blocks/editor.scss

```
@@ -121,7 +121,7 @@ $icon-background-color: #fff;
 
 	// Remove paid-block-media-placeholder from being a flexbox child
 	// when the cover block has a background set
-	.wp-block-cover &:not( :only-child ) {
+	.wp-block-cover :not( .is-placeholder ) {
```

I see a couple of issues with the way this CSS selector is written:

- the `:not( .is-placeholder )` selector should be applied to `.wp-block-cover` (i.e. no space before the `:not`)
- we should add a `&` at the end of the selector, to make sure that the styles apply to `.paid-block-media-placeholder`


```suggestion
	.wp-block-cover:not( .is-placeholder ) & {
```